dcicutils/s3_utils.py

```diff
@@ -320,17 +313,16 @@ def get_key(self, keyfile_name: str = 'access_key_admin'
                                       Key=keyfile_name,
                                       SSECustomerKey=os.environ['S3_ENCRYPT_KEY'],
                                       SSECustomerAlgorithm='AES256')
-        akey: Union[bytes, str] = response['Body'].read()
-        if type(akey) == bytes:
-            akey: str = akey.decode()
+        body_data: Union[bytes, str] = response['Body'].read()
+        auth_text: str = body_data.decode() if type(body_data) == bytes else body_data
         try:
-            return json.loads(akey)
+            return json.loads(auth_text)
         except (ValueError, TypeError):
             # TODO: Is there really a use case for this returning non-JSON data? If so, can we make a different
             #       function that does this? Because this is just going to return garbage when peopla are expecting
             #       key dictionaries and we really should raise an error in that case. -kmp 1-Mar-2023
             # maybe its not json after all
-            return akey
+            return auth_text
```

Comment:
I renamed some variables and did some minor code reshaping above, but did not intentionally change the logic.